community/index.md

```diff
@@ -1,9 +1,18 @@
 ---
 title: Community
 ---
-# Community
+# {{ page.title }}
 
-### Contact Us
+## Troubleshooting
+
+Coming soon.
+
+## [GCP Resource Coverage]({% link _docs/quickstarts/inventory/index.md %}#gcp-resource-coverage)
```

Comment:
I was under the impression that Community is social -- not sure how the Resource Coverage fits here.